### ISSUE TITLE
Is it possible to wrap document.body (or any children) in a React.Wrapper element?

### ISSUE THREAD

**Issue by matthieu-beteille** [enzymejs/enzyme#2389]
**Is your feature request related to a problem? Please describe.**
I am on react 15, a library that I use render Popover elements in the body at the top level.

**Describe the solution you'd like**
I'd like to do something like;

```Typescript
const popover = document.body.querySelector(".my-popover-class");

// this does not work, but it would be great
const popoverWrapper = new ReactWrapper(popover);

const myComponent = popoverWrapper.find(MyReactComponentNestedInThePopover);
```

Is this currently possible?

**Comment by ljharb** [comment]
No, it's not. enzyme's shallow and mount are for wrapping react elements, not DOM elements.

Prior to React officially supporting "portals", there's no really good solution for you. You could use enzyme's `render(popover)`, but that's just cheerio wrapping the DOM.

**Comment by matthieu-beteille** [comment]
thanks for the quick answer :+1:
